src/lib.rs

```diff
@@ -2,9 +2,26 @@
 //!
 //! `ayb` makes it easy to create, host, and share embedded databases like SQLite and DuckDB.
 
+use prettytable::{format, Table};
+
 pub mod ayb_db;
 pub mod email;
 pub mod error;
 pub mod hosted_db;
 pub mod http;
 pub mod templating;
+
+pub trait FormatResponse {
```

Comment:
Should this live in its own module like `formatting`?

And should we name it something like `RelationalFormatter` or `TabularFormatter` since it formats relations/tables?

Reply:
I don't feel strongly, I'll move it to its own module. I like `TabularFormatter` :)